tracker_test.go

```diff
@@ -79,24 +99,24 @@ func TestTrackerListItemsNotNil(t *testing.T) {
 			t.Error("Returned Tracker found nil")
 		}
 	}
-	test.DeleteTrackerOK(t, nil, nil, &controller, item1.ID)
-	test.DeleteTrackerOK(t, nil, nil, &controller, item2.ID)
+	test.DeleteTrackerOK(t, nil, nil, &controller, *item1.Data.ID)
+	test.DeleteTrackerOK(t, nil, nil, &controller, *item2.Data.ID)
```

Comment:
Use `gormsupport.DeleteCreatedEntities` as a deferred call. There are many examples in other files.